vector-config/src/main/res/values/config-settings.xml

```diff
@@ -49,6 +49,8 @@
     <bool name="settings_timeline_show_live_sender_info_default">false</bool>
     <bool name="settings_labs_rich_text_editor_visible">true</bool>
     <bool name="settings_labs_rich_text_editor_default">false</bool>
+    <bool name="settings_labs_enable_voice_broadcast_visible">false</bool> <!-- Note: also defined in values-v29 -->
```

Comment:
If you keep this, I would rather comment like this:
```suggestion
    <bool name="settings_labs_enable_voice_broadcast_visible">false</bool> <!-- Note: this value must always be false. See in values-v29/config-settings to toggle the visibility of this flag. -->
```

But I would definitely go for the test at runtime, to avoid any mistakes.

So this could be changed to 

 ```suggestion
    <!-- Note: on device with API < 29, setting this value to true has no effect. -->
    <bool name="settings_labs_enable_voice_broadcast_visible">false</bool>
```